artemis-pkg-copy/src/api/api.toit

```diff
@@ -59,6 +59,9 @@ interface ArtemisService:
   controller-open --mode/int -> int
   static CONTROLLER-OPEN-INDEX /int ::= 6
 
+  controller-open --mode/int --force-recovery-contact/bool -> int
```

Comment:
I'm not super keen on the "contact" part of this. It feels a bit weak, because we're also going to try recover (download new broker info, etc.). At the same time, it feels a bit too specific, because there is no guarantee that we can contact a recovery server.

Reply:
changed API.